src/algebra/monoid_algebra/basic.lean

```diff
@@ -548,14 +548,23 @@ lemma single_one_comm [comm_semiring k] [mul_one_class G] (r : k) (f : monoid_al
 by { ext, rw [single_one_mul_apply, mul_single_one_apply, mul_comm] }
 
 /-- `finsupp.single 1` as a `ring_hom` -/
-@[simps] def single_one_ring_hom [semiring k] [monoid G] : k →+* monoid_algebra k G :=
+@[simps] def single_one_ring_hom [semiring k] [mul_one_class G] : k →+* monoid_algebra k G :=
 { map_one' := rfl,
   map_mul' := λ x y, by rw [single_add_hom, single_mul_single, one_mul],
   ..finsupp.single_add_hom 1}
 
+/--  A multiplicative homomorphism `f : G →* H` between two monoids induces a ring
+homomorphism `monoid_ring_hom_map f : monoid_algebra k G →+* monoid_algebra k H`. -/
+def monoid_ring_hom_map (k : Type*) {H F : Type*} [semiring k] [monoid G] [monoid H]
```

Comment:
```suggestion
@[simps]
def monoid_ring_hom_map (k : Type*) {H F : Type*} [semiring k] [monoid G] [monoid H]
```
assuming that generates something like `monoid_ring_hom_map _ _ _ = map_domain _ _`

Reply:
Do you mean `simps`? Use `simps?` to find out

Reply:
Yes, I meant `simps`!  Where do I simply type `simps?`

Reply:
Yep, just add the question mark